core-web/libs/sdk/angular/next/components/dotcms-block-editor-renderer/blocks/video.components.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Input } from '@angular/core';
+
+import { DotCmsClient } from '@dotcms/client';
+import { ContentNode } from '@dotcms/uve/internal';
+
+@Component({
+    selector: 'dotcms-block-editor-renderer-video',
+    standalone: true,
+    template: `
+        <video
+            [controls]="true"
+            preload="metadata"
+            [poster]="posterUrl"
+            [width]="attrs?.['width']"
+            [height]="attrs?.['height']">
+            <track default kind="captions" srclang="en" />
+            <source [src]="srcUrl" [type]="attrs?.['mimeType']" />
+            Your browser does not support the
+            <code>video</code>
+            element.
+        </video>
+    `
+})
+export class DotCMSBlockEditorRendererVideoComponent {
+    @Input() attrs!: ContentNode['attrs'];
+
+    private client = DotCmsClient.instance;
```

Comment:
We need to avoid client dependencies. We can write a proxy in the example to handle assets from dotCMS, similar to what we do in ReactJS